### ISSUE TITLE
busser-nose2 requires pip to be installed in the system

### ISSUE THREAD

**Issue by lieryan** [thedrow/busser-nose2#1]
During plugin installation in kitchen, busser suggested that the nose2 plugin was installed successfully, even though it hasn't. busser-nose2 uses pip to install nose2, but in many systems pip isn't preinstalled.

**Comment by thedrow** [comment]
I think that it's a reasonable requirement.

**Comment by lieryan** [comment]
I would disagree that it is a reasonable requirement, Busser was supposed to be "a test setup and execution framework designed to work on remote nodes whose system dependencies cannot be relied upon, except for an Omnibus installation of Chef." My use case is in using nose2 so that I can write server config tests in python rather than ruby as I am more familiar with that but not necessarily for deploying a python application, so I don't really want the entire python development/deployment suites installed in the final system.

Putting that aside though, even if we are to agree that it is a reasonable requirement, I think that having at least an error message that indicates that busser setup didn't succeed would be reasonable. As a beginner of kitchen/busser testing, I was scrambling around for a long while trying to figure out why my tests wasn't running correctly as nothing in the error message clued in that nose2 is missing.

Is there a way to change https://github.com/thedrow/busser-nose2/blob/master/lib/busser/runner_plugin/nose2.rb#L27 so it does not ignore the return code when pip is missing from the system?
